Jenkinsfile

```diff
@@ -23,6 +30,9 @@ pipeline {
                     git submodule foreach --recursive git reset --hard
                     git submodule foreach --recursive git clean -fdx
                     git submodule status
+                    export commit_sha=$(git submodule status | grep 'pmm-managed' | awk -F ' ' '{print $1}')
+                    curl -s https://api.github.com/repos/percona/pmm-managed/commits/${commit_sha} | grep 'name' | awk -F '"' '{print $4}' | head -1 > OWNER
```

Comment:
Maybe we can get owner using the following command: git --no-pager show -s  --format="%an <%ae>" commit_id

Reply:
btw, we can get last commit author for submodules using this command 
```git submodule foreach 'git log -1 --pretty=format:"%ae %an"'```

Reply:
I think that would return me github userid for most of the commits, I need names which is returned by the API call, @hors  what you suggest is good if we need info for main repo and not submodule.